completions/_vagrant

```diff
@@ -14,6 +14,15 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+	        if [ "$2" = "init" ]; then
```

Comment:
Actually further thinking on it, you probably don't need this at all since there's not really a different scenario to get boxes under.  Something like this should probably do better.

```zsh
local -a boxes
boxes=(
	$(vagrant box list | cut -d " " -f 1)
)

_describe -t commands "vagrant boxes" boxes -V1
```